.github/workflows/destroy_aws_environment.yml

```
@@ -22,30 +22,172 @@
 
   manual-approval:
     needs: [destroy-plan]
     runs-on: ubuntu-latest
-    if: ${{ !inputs.plan_only }}
+    if: ${{ inputs.plan_only == false && inputs.environment != 'Development' }}
     steps:
-    - name: Placeholder - Manual approval step
-      run: |
-        echo "This step is empty - manual approval functionality not implemented on main branch"
-        echo "Would wait for approval to destroy ${{ inputs.environment }}"
+      - name: Wait for approval to destroy ${{ vars.ENV_NAME }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)